Project/frontend/src/components/UploadPage/index.tsx

```diff
@@ -87,17 +87,19 @@ function UploadPage() {
 
   return (
     <Stack flex={1} justifyContent={'center'} alignItems={'center'} spacing={2}>
-      <Typography
-        variant="h6"
-        className="title"
-        sx={{ color: (theme) => theme.palette.text.secondary }}
-      >
-        Upload a document to generate the graph
-        <Tooltip title={<Typography>{hintText}</Typography>}>
+      <div className='upload_info'>
```

Comment:
why??

Reply:
The info icon is not aligned with the info text. So fixed it!